README.md

```diff
@@ -71,76 +79,109 @@ spec:
-  Everything is served on port 8080: the `/auth` and `/hub` routes are proxied to their services, and all other routes serve the UI bundle where they are handled by react-router.
-- In development mode, webpack-dev-server plays the role of UI server and Express plays the role of proxy server only. This allows webpack-dev-server to provide development features such as auto-reload.
-  webpack-dev-server serves the UI on port 9000 and forwards the `/auth` and `/hub` routes to port 8080 for Express to handle.
+- In **production** mode, Express (Node.js) plays the role of both UI server and proxy server
+  (using http-proxy-middleware). Everything is served on port **8080**. The `/auth` and `/hub`
+  routes are proxied to their services. All other routes serve the UI bundle where they are
+  handled by react-router.
 
-The Express server (server/setupProxy.js) uses by default the environment variables TACKLE_HUB_URL and SSO_SERVER_URL to determine the backend endpoints.
-If no env. variables are defined, the server then listens on ports 9001 (SSO), 9002 (Application inventory and controls).
+- In **development** mode, webpack-dev-server plays the role of UI server and Express plays
+  the role of proxy server only. This allows webpack-dev-server to provide development features
+  such as hot reload. The webpack-dev-server serves the UI on port **9000**. The `/auth` and `/hub`
+  routes are forwarded to port **8080** for Express to handle.
 
-In which case the port forwarding must be activated to route Tackle Keycloack (SSO), Tackle Hub requests.
-To set-up kubectl port forwarding Tackle2 services to localhost, open a terminal and run each following command separately:
+The Express [server/src/setupProxy.js](server/src/setupProxy.js) proxies use the environment
+variables `TACKLE_HUB_URL` and `SSO_SERVER_URL` to define the backend endpoints:
 
-```
+- If the Tackle Hub variable `TACKLE_HUB_URL` is not defined, the URL `http://localhost:9002` is
+  used by default.
+
+- If the Tackle Keycloak (SSO) variable `SSO_SERVER_URL` is not defined, the URL
+  `http://localhost:9001` is used by default.
+
+To enable running the UI outside the cluster, port forwardings must be activated to route
+the Tackle Keycloak (SSO) and Tackle Hub requests to the services on the cluster. To setup
+the kubectl port forwards needed, open a terminal and run each following command separately:
+
+```sh
 $ kubectl port-forward svc/tackle-keycloak-sso -n konveyor-tackle 9001:8080
 $ kubectl port-forward svc/tackle-hub -n konveyor-tackle 9002:8080
-$ kubectl port-forward svc/tackle-pathfinder -n konveyor-tackle 9003:8080
 ```
 
-That's exactly what `start:dev` does by port forwarding all tackle2 services for us.
+The `kubectl port-forward` commands need to remain running for the ports to be available. Use
+the script `npm run port-forward` to easily start the forwards. The script `npm run start:dev`
```

Comment:
Might be worth moving 
> `The script npm run start:dev will also to setup port forwarding to all tackle2 services when starting the dev server.`
to the beginning of this section. It reads as though the user must do this additional step. Just my 2 cents.